src/middlewares/task.middleware.js

```diff
@@ -270,3 +276,9 @@ const getDevServerCommand = (
       throw new Error('Unrecognized project type: ' + projectType);
   }
 };
+
+const sendCommandToProcess = (child: any, command: string) => {
+  // Commands have to be suffixed with '\n' to signal that the command is
+  // ready to be sent. Same as a regular command + hitting the enter key.
+  child.stdin.write(`${command}\n`);
```

Comment:
@superhawk610 At the moment, I'm merging windows-support & npm-to-yarn branch (not pushed yet) and there I've noticed that ejecting is not working with-out error message and it seems that it is happening at line 283. The error message is like in this [issue](https://github.com/nodejs/node/issues/18446).

Telling that the socket is already closed. Not exactly sure what happens here but it could be related that we're not sending `end` after writing to stdin.

Sending `end` can also be problematic because it could close the process.

I'll try to eject at Bennys windows-support branch so I know if it was working there. Maybe this could also be related to my setup:
Windows 10
npm -v  5.6.0
node -v v8.11.1

The app is ejected afterwards but there is this error message in console.

Reply:
OK, not sure what it was but after removing `node_modules` and `yarn.lock` + reinstalling. Ejecting is working like it is.